tests/pages/regions/ckeditor.py

```diff
@@ -31,33 +38,104 @@ def ready(self):
     def content_source(self):
         return self.selenium.execute_script(self.CKEDITOR_CONTENT_QUERY)
 
-    def draft_content(self, base_url):
+    def draft_slug(self, draft_url):
         # remove domain from base url
-        url = urlparse(base_url)
-        if url.path.startswith('/en-US/docs/new'):
-            draft_name = 'draft/new'
-        else:
-            url_strip_edit = url.path.replace('$edit', '')
-            # check if this is a regular edit page or translation
-            if not url.path.startswith('/en-US/'):
-                draft_name = 'draft/translate{}/'.format(url_strip_edit)
-            else:
-                draft_name = 'draft/edit' + url_strip_edit
-        return self.selenium.execute_script(
+        # TODO this function does not work for drafts for new translations
+        url = urlparse(draft_url)
+        url_strip_edit = url.path.replace('$edit', '')
+        return 'draft/edit' + url_strip_edit
+
+    @property
+    def is_draft_container_displayed(self):
+        return self.find_element(
+            *self._draft_container_locator
+        ).is_displayed()
+
+    @property
+    def is_draft_old_displayed(self):
+        return self.find_element(
+            *self._draft_old_locator
+        ).is_displayed()
```

Comment:
This will raise an exception if the element doesn't exist, which is being tested in ``test_article_new``.  I'd add an import at the top:

    from selenium.common.exceptions import NoSuchElementException

and then write it like:

    def is_draft_old_displayed(self):
        try:
            return self.find_element(
                *self._draft_old_locator
            ).is_displayed()
        except NoSuchElementException:
            return False